docs/arch/adr-11.md

```diff
@@ -0,0 +1,48 @@
+# ADR 11: Rewriting Markdownz to use remark
+
+Created: December 19, 2018
+
+## Context
+
+Markdown can help prevent XSS type security vulnerabilities and is generally safer to use than HTML for user submitted content on the web. Panoptes-Front-End uses markdown through out the entire application. Currently we support an in-house markdown renderer, [markdownz](https://github.com/zooniverse/markdownz), that uses  [markdown-it](https://github.com/markdown-it/markdown-it). The library markdown-it is mature and has several plug-ins available for it that we've added to markdownz as well as some of our own customizations.
```

Comment:
Minor point but I'm not sure about the opening sentence here. Markdown gives us more control over what HTML users can publish on our site, I guess. I don't know if I would say that Markdown itself prevents vulnerabilities.